databox/api/src/Metadata/MetadataNormalizer.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Metadata;
+
+use PHPExiftool\Driver\Metadata\Metadata;
+use PHPExiftool\Driver\Metadata\MetadataBag;
+use PHPExiftool\Driver\Value\Binary;
+
+class MetadataNormalizer
+{
+    /**
+     * normalize metadata from metadataManipulator bundle (for File.metadata).
+     */
+    public function normalizeToArray(MetadataBag $bag): array
+    {
+        $a = [];
+
+        /** @var Metadata $meta */
+        foreach ($bag as $meta) {
+            $vMeta = $meta->getValue();
+
+            // skip "declared-binary" and "binary-not-declared-binary" data
+            if ($vMeta instanceof Binary) {
+                continue;
+            }
+            try {
+                if (!json_encode($vMeta->asString())) {
```

Comment:
Why this test?

Reply:
exiftool (so metadata-manipulator lib also) reports some binary data as "string", while it's raw binary - that will fail json_encode -.
Binary metadata includes heavy blobs like thumbnail, color curves, icc profiles, ... 
One may hex-encode before json, but this creates useless (?) huge values into file.metadata.
Can be fixed later if we want to get binaries like "thumbnail", "